Description: add a simple CMake build system for installing
 The CMakeLists.txt is a simplified version of upstream's one.
Author: Pino Toscano <pino@debian.org>
Forwarded: not-needed

--- /dev/null
+++ b/CMakeLists.txt
@@ -0,0 +1,8 @@
+project(kdesrc-build)
+
+find_package(KDE4 REQUIRED)
+include(KDE4Defaults)
+
+install(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/kdesrc-build DESTINATION ${BIN_INSTALL_DIR})
+install(FILES kdesrc-buildrc.xml DESTINATION ${DATA_INSTALL_DIR}/katepart/syntax)
+
